Documentation/operations/troubleshooting.rst

```diff
@@ -148,6 +148,38 @@ Observing Flows with Hubble
 Hubble is a built-in observability tool which allows you to inspect recent flow
 events on all endpoints managed by Cilium.
 
+Ensure Hubble is running correctly
+----------------------------------
+
+To ensure the Hubble client can connect to the Hubble server running inside
+Cilium, you may use the ``hubble status`` command from within a Cilium pod:
+
+.. code-block:: shell-session
+
+   $ hubble status
```

Comment:
This will not work as documented with hubble v0.8 correct? I believe we will need:
```suggestion
   $ hubble status --server unix:///var/run/cilium/hubble.sock
```

Reply:
No, this is not required as we set `HUBBLE_SERVER=unix:///var/run/cilium/hubble.sock` in the cilium container (ee #15983).